api/pkg/guard/guard.go

```diff
@@ -62,29 +61,30 @@ func CheckMember(namespace *models.Namespace, id string) (*models.Member, bool)
 	return &found, true
 }
 
-// GetRoleCode converts a models.Member's role string to a role code. If the role is not found in Roles, it returns -1.
+// GetRoleCode converts a models.Member's role string to a role code. If the role is not found in Roles, it returns InvalidRole.
 func GetRoleCode(role string) int {
 	code, ok := Roles[role]
 	if !ok {
-		// return -1 when member role is not valid.
-		return -1
+		// return InvalidRole when member's role is not valid.
+		return InvalidRole
```

Comment:
Using the constant `InvalidRole` to represent the value when the member's role does not exist.

Reply:
Seems better